wadas/domain/operation_mode.py

```diff
@@ -93,6 +95,32 @@ def _detect(self, cur_image_path):
         self.last_detection = detected_img_path
         return results, detected_img_path
 
+    def _format_classified_animals_string(self, classified_animals):
+        # Prepare a list of classified animals to print in UI
+        self.last_classified_animals_str = ""
+        for animal in classified_animals:
+            last = animal["classification"][0]
+            if not self.last_classified_animals_str:
+                self.last_classified_animals_str = self.last_classified_animals_str + last
+            else:
+                self.last_classified_animals_str = self.last_classified_animals_str + ", " + last
```

Comment:
self.last_classified_animals_str = ", ".join(animal["classification"][0] for animal in classified_animals)